test/wiki-links.txt

```diff
@@ -37,3 +37,48 @@ Wiki links are prioritized over regular links.
 <p><x-wikilink data-target="foo">foo</x-wikilink>(foo.jpg)</p>
 ````````````````````````````````
 
+Something that may look like a wiki link at first, but turns out not to be,
+is recognized as a normal link.
+
+```````````````````````````````` example
+[[foo]
+
+[foo]: /url
+.
+<p>[<a href="/url">foo</a></p>
+````````````````````````````````
+
+Escaping the opening `[`, we should get a normal link.
+
+```````````````````````````````` example
+\[[foo]]
+.
+<p>[<a href="/url">foo</a>]</p>
```

Comment:
By the way, here is a failing test case. I'm not sure what should be the proper behavior here. I was hoping you had some input.

Reply:
That `[foo]` inside is not a link because it has no link destination and there is no corresponding link reference definition, so it renders correcly to `<p>[[foo]]</p>`.

If I add the definition:
```
[foo]: /url

\[[foo]]
```

then it outputs what you expect.